scripts/docker/gatkbase/Dockerfile

```diff
@@ -72,4 +72,6 @@ RUN mkdir $DOWNLOAD_DIR && \
     bash $DOWNLOAD_DIR/miniconda.sh -p $CONDA_PATH -b && \
     rm $DOWNLOAD_DIR/miniconda.sh && \
     conda clean -afy && \
+    conda config --set auto_update_conda false && \
```

Comment:
Will hopefully ensure that devs experimenting with updates to the environment within the Docker don't inadvertently update to a different conda version, which might otherwise introduce discrepancies.